Rubberduck.Parsing/VBA/RubberduckParser.cs

```diff
@@ -89,15 +89,13 @@ public void Parse()
         {
             if (!_state.Projects.Any())
             {
-                foreach (var project in _vbe.VBProjects.Cast<VBProject>())
+                foreach (var project in _vbe.VBProjects.UnprotectedProjects())
```

Comment:
Hopefully this extension method will help prevent locked projects biting me again.